### DISCUSSION TITLE
Help me find the implementation part for dynamic query search

### DISCUSSION THREAD

**Discussion by SAISRIHARSHAAMBATI** [grafana/grafana#63319] [General]
how dynamic web query search is implemented in Grafana and also how to go through this GitHub source(where can I find the implementation part for dynamic query search that is happening)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
